azure-pipelines.yml

```diff
@@ -60,16 +60,12 @@ jobs:
 
   - job: 'DockerSlim'
     pool:
-      vmImage: ubuntu-16.04
+      vmImage: ubuntu-18.04
 
     steps:
-      - bash: |
-          docker pull renovate/renovate:slim
-        displayName: Pull old slim renovate image as cache
-
       - bash: |
           export DOCKER_BUILDKIT=1
-          docker build --cache-from renovate/renovate:slim -t renovate-test -f Dockerfile.slim .
```

Comment:
How come to remove it?

Reply:
It slows down, because the cache does not work as expected